sidekiq_prometheus.gemspec

```diff
@@ -28,7 +28,8 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "standard"
 
   spec.add_runtime_dependency "prometheus-client", ">= 2.0"
-  spec.add_runtime_dependency "rack", "< 3.0"
+  spec.add_runtime_dependency "rack", "~> 3.0"
```

Comment:
does this need to be `~> 3.0` or could it be `'>= 2.0', '< 4'` As far as possible it's so valuable for consumers to leave the library's requirements wide open

Reply:
Here's what I ran in to when trying to support more open requirements

`Rack::Handler` was removed from the main rack codebase and moved to the `rackup` (`Rackup::Handler`). I set up different gemfiles in the matrix to test against Rack 2 and Rack 3. The `rackup` gem requires Rack >3.0 so testing against Rack 3 was fine. But then when running against Rack 2 the dependencies couldn't resolve because rack was set to v2 but it was also trying to load rackup. 

Leaving `rackup` out and requiring the users of the gem to explicitly require `rackup` if they wanted to use Rack 3 was an option, but not a great one.

Reply:
Ah gotcha, there's a similar issue with Faraday 2->3 and the faraday-middleware gem